packages/cells/test/date-picker-cell.test.tsx

```diff
@@ -156,52 +156,55 @@ describe("editor", () => {
 
         const mockCellOnChange = jest.fn();
         const result = render(<Editor isHighlighted={false} value={getMockDateCell()} onChange={mockCellOnChange} />);
-        const input = await result.findByTestId("test-id");
-        expect(result.findByTestId("test-id")).not.toBeUndefined();
+        const input = await result.findByTestId("date-picker-cell");
+        expect(result.findByTestId("date-picker-cell")).not.toBeUndefined();
         fireEvent.change(input, { target: { value: "" } });
         expect(mockCellOnChange).toHaveBeenCalledTimes(1);
         expect(mockCellOnChange).toBeCalledWith({
             kind: GridCellKind.Custom,
             allowOverlay: true,
             copyData: "4",
+            readonly: false,
             data: {
                 kind: "date-picker-cell",
                 // should be undefined since value is ''
                 date: undefined,
                 displayDate: new Date("2023-02-06T04:47:44.584Z").toISOString(),
                 format: "time",
-                readonly: false,
             },
         });
     });
 });
 
 describe("onPaste", () => {
     it.each([
-        ["2023-02-06T04:47:44.584Z"],
-        ["1995-12-17T03:24:00"],
-        ["Sun Dec 17 1995 03:24:00 GMT"],
-        [new Date(1995, 11, 17)],
-        [100],
-        [-1],
-    ])("correctly returns a value when onPaste is called with valid value: %p", (input: string | number | Date) => {
+        ["2023-02-06T04:47:44.584Z", "2023-02-06T04:47:44.584Z"],
+        ["1995-12-17", "1995-12-17T00:00:00.000Z"],
+        ["1995-12-17T03:24:00Z", "1995-12-17T03:24:00.000Z"],
+        ["1995-12-17T03:24:00", "1995-12-17T02:24:00.000Z"],
+        ["Sun Dec 17 1995 03:24:00 GMT", "1995-12-17T03:24:00.000Z"],
+        [new Date(1995, 11, 17), "1995-12-16T23:00:00.000Z"],
+        [100, "1970-01-01T00:00:00.100Z"],
+        [-1, "1969-12-31T23:59:59.999Z"],
+        [-1, "1969-12-31T23:59:59.999Z"],
+    ])("correctly interprets pasted value %p as %p", (input: string | number | Date, expected: string) => {
         // @ts-ignore
         const { date } = renderer.onPaste(input, {});
-        expect(date).toStrictEqual(new Date(input));
+        expect(date.toISOString()).toStrictEqual(expected);
```

Comment:
Is there a specific reason to cast it to `isoString()`. I thought about that but just thought reading the test cases was harder and new Date(input) without the isoString is still the same date object, just one level deeper which it feels like we don't need?

Reply:
Hmm, yes. So I added the expected cases above to make it slightly more explicit. E.g. if the timezone is not specific, the date will be converted to UTC based on your local timezone: `"1995-12-17T03:24:00", "1995-12-17T02:24:00.000Z"]`. In that case, both solutions would work fine: 
`expect(date).toStrictEqual(new Date(expected));`
or
`expect(date.toISOString()).toStrictEqual(expected);`. But - if I remember correctly - the second one gives slightly better error messages if something goes wrong.